### DISCUSSION TITLE
2023/09/29/ahskqfntuduz

### DISCUSSION THREAD

**Discussion by giscus** [kaochenlong/kaochenlong-discuss#1] [General]
# 2023/09/29/ahskqfntuduz

在前篇文章我跟大家介紹了 senior 工程師的共通特質以及 junior 工程師的自我修練的建議，希望對大家有些幫助。但如果你不是軟體工程師，不知道在哪裡看到招生廣告點進來的，或是剛好正在考慮想要轉職軟體工程師的話，有一些過去以及我們自己帶學生的經驗可供你參考。同樣，如果懶的看文字，也可以看影片配雞排：你為什麼...

http://localhost:3000/2023/09/29/ahskqfntuduz.html

**Comment by kaochenlong** [comment]
hello world

```js
console.log(123);
```
